eStore/Scenes/Catalog/CatalogContentView/CategoryTableViewDataSource.swift

```diff
@@ -9,6 +9,7 @@
 import UIKit
 
 class CategoryTableViewDataSource: NSObject, UITableViewDataSource {
+    private let themeManager = ThemeManagerImpl() // TODO: dependency injection
```

Comment:
How can I inject my service to `DataSource`? It's accessible only via presenter

Reply:
Well you can just create `theme` property here, apply it as you do on `cellForRow`, but in this case you should inject this property through the view from the presenter. I think this is okay as the theme itself is kind of a presentation model which should be propagated as a regular model to view to display something. Just think of it as some sort of data / model whatever, no need to create a service (ThemeManager) here